src/signer_node/node_parameters.rs

```diff
@@ -65,4 +66,53 @@ impl<T: TapyrusApi> NodeParameters<T> {
             share_count: n.clone(),
         }
     }
+
+    fn sort_publickey(pubkeys: &mut Vec<PublicKey>) {
+        pubkeys.sort_by_key(|p| {
+            let serialized = p.key.serialize();
+            let hex = hex::encode(&serialized[..]);
+            hex
```

Comment:
Is encoding required?
Does returning [u8] or Vec<u8> occur any problems?

Reply:
I think it's true. I will try it.

Reply:
updated #7b4fccd